sources/engine/Stride.Graphics/Vulkan/PipelineState.Vulkan.cs

```diff
@@ -433,15 +431,23 @@ private unsafe VkPipelineShaderStageCreateInfo[] CreateShaderStages(PipelineStat
                     {
                         sType = VkStructureType.PipelineShaderStageCreateInfo,
                         stage = VulkanConvertExtensions.Convert(stages[i].Stage),
-                        pName = entryPointPointer,
+                        pName = entryPointPointer
                     };
-                    vkCreateShaderModule(GraphicsDevice.NativeDevice, shaderBytecode.Data, null, out nativeStages[i].module);
+                    vkCreateShaderModule(GraphicsDevice.NativeDevice, shaderBytecode.Data, allocator: null, out nativeStages[i].module);
                 }
             };
 
             return nativeStages;
         }
 
+        //
+        // Reads the ShaderInputBytecode structure from a byte[].
+        //
```

Comment:
Triple slash and `<summary>` missing

Reply:
Sorry about this one. It's not a triple slash comment with summary because I tend to not write those for local functions (as, at least for me, IntelliSense does not show me info about local functions).

I can change it to triple-slash, or make it a single comment, or remove it.

Reply:
That's weird, intellisense should definitely show them. I think the name of the function speaks for itself, but if you want to keep the comment, yeah it should be inside a summary block. Thanks !

Reply:
I've retested this on a laptop and there IntelliSense also does not show info for local methods. Does it shows for you?
Anyway, I'll make a full `<summary>` comment.

Reply:
Yeah, VS2022:
![image](https://github.com/stride3d/stride/assets/5742236/d99ad742-165f-4cf3-9102-a3907fdf1a42)
Maybe some sort of low perf mode for large solutions ? Though that seems very unlikely.

Reply:
Visual Studio do show IntelliSense for those methods (class methods, both static and instance). The local functions (inside of a method) however (as the function in the PR) do show no summary nor parameter info nor any other info.
Maybe Rider do? I haven't tested VSCode.
I tend to use a lot of mini-functions in the local scope to remove repetition and aid in legibility, and the lack of IntelliSense info is a bit of a bummer.